### ISSUE TITLE
when will a new version with the opensearch output be released?

### ISSUE THREAD

**Issue by RodrigoDornelles** [influxdata/telegraf#14332]
### Please direct all support questsions to slack or the forums. Thank you.

Apparently it is in the main branch or opensearch, but it is still not available in the releases and containers, when what would be available?

 * <https://github.com/influxdata/telegraf/tree/master/plugins/outputs/opensearch>
 * <https://github.com/influxdata/telegraf/tree/v1.28.5/plugins/outputs/opensearch>
 * <https://github.com/influxdata/telegraf/pull/11958>
 * <https://github.com/influxdata/telegraf/milestone/94>
 * <https://github.com/influxdata/telegraf/issues/14334>
 * <https://github.com/influxdata/telegraf/issues/9414>
 * <https://github.com/influxdata/telegraf/issues/11922>
 
 ```
I! Loading config: /etc/telegraf/telegraf.conf
E! error loading config file /etc/telegraf/telegraf.conf: error parsing OpenSearch array, undefined but requested output: OpenSearch
 ```
 
Basically I wanted to know if something is missing for opensearch in telegraf **1.29.0**. I can contribute with something because it is a resource that I really need.

**Comment by srebhan** [comment]
@RodrigoDornelles the basic rule is for versions `v1.x.y` is that `v1.x.y+1` only contain _bug-fixes_, documentation changes etc but _NO_ new features. A release `v1.x+1.0` will then _also_ contain new _features_ (besides bug-fixes, etc).

So for your question: The mentioned output is a new plugin i.e. a feature and is thus released with `v1.29.0` scheduled for 2023-12-11. You can also find out yourself by clicking on PR #11958, on the right hand side it shows the "milestone". If you click on the version you will see the milestone and the related "due date" which is the planned release date.
